open-banking-accelerator/components/event-notifications/com.wso2.openbanking.accelerator.event.notifications.service/src/main/java/com/wso2/openbanking/accelerator/event/notifications/service/constants/EventNotificationConstants.java

```diff
@@ -32,6 +32,12 @@ public class EventNotificationConstants {
     public static final String OPEN = "OPEN";
 
     //Response Status
+    public static final int NOT_FOUND_CODE = 404;
+    public static final int OK_CODE = 200;
+    public static final int CREATED_CODE = 201;
+    public static final int BAD_REQUEST_CODE = 400;
+    public static final int NO_CONTENT_CODE = 204;
+    public static final int INTERNAL_SERVER_ERROR_CODE = 500;
```

Comment:
Can't we get these from [apache commons](https://hc.apache.org/httpclient-legacy/apidocs/org/apache/commons/httpclient/HttpStatus.html)?

Reply:
Fixed in https://github.com/wso2/financial-open-banking/pull/3/commits/a31eef6176e5003bd359e183aae45a90a5874f2b